python/ray/tune/tests/test_ray_trial_executor.py

```diff
@@ -615,9 +622,12 @@ def train(config):
 
         register_trainable("resettable", train)
 
-        trial1 = Trial("resettable", placement_group_factory=pgf1)
-        trial2 = Trial("resettable", placement_group_factory=pgf1)
-        trial3 = Trial("resettable", placement_group_factory=pgf2)
+        trial1 = _make_trial("resettable", placement_group_factory=pgf1)
+        trial1.create_placement_group_factory()
+        trial2 = _make_trial("resettable", placement_group_factory=pgf1)
+        trial2.create_placement_group_factory()
+        trial3 = _make_trial("resettable", placement_group_factory=pgf2)
+        trial3.create_placement_group_factory()
```

Comment:
```suggestion
        trial1 = _make_trial("resettable", placement_group_factory=pgf1)
        trial2 = _make_trial("resettable", placement_group_factory=pgf1)
        trial3 = _make_trial("resettable", placement_group_factory=pgf2)
```

shouldn't be needed with `make_trial`?